### PR TITLE
Setting platform dependent docker-compose executable name

### LINKED ISSUE #416: Local Docker compose impossible on Windows
Hi

In 1.4.1, the implementation for the `LocalDockerCompose` class has [a constant specifying the executable name of local Docker Compose](https://github.com/testcontainers/testcontainers-java/blob/testcontainers-1.2.1/core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java#L444). 

This is hard-coded to `docker-compose` which is fine on *nix / OS X, but fails on Windows, which is expecting `docker-compose.exe`.

It might be nice if this auto-detected Windows and used the `.exe` version, or even just tried both in turn...

### PR DESCRIPTION
**Pull request by barrycommins**
Fixes #416

### CODE REVIEW

File: core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java [hunk 1]
```diff
@@ -464,7 +465,15 @@ public void invoke() {
     /**
      * Executable name for Docker Compose.
      */
-    private static final String COMPOSE_EXECUTABLE = "docker-compose";
```
Comment (bsideup) [inline core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java]: I would keep final and use something like `SystemUtils.IS_OS_WINDOWS ? "docker-compose.exe" : "docker-compose"`

Comment (barrycommins) [inline core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java]: Hi @bsideup .
I was trying to stay consistent with this (although I missed that it was still final) : 
https://github.com/testcontainers/testcontainers-java/blob/32b655190ddc7d2b24d136370cf4ca562a536903/core/src/main/java/org/testcontainers/utility/DockerMachineClient.java#L24-L31

I agree that your version is cleaner. I'll update the PR with that change.

File: core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java [hunk 2]
```diff
@@ -464,7 +465,7 @@ public void invoke() {
...
+    private static final String composeExecutable = SystemUtils.IS_OS_WINDOWS ? "docker-compose.exe" : "docker-compose";
```
Comment (bsideup) [inline core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java]: Could you please also revert the variable name change?

File: core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java [hunk 3]
```diff
@@ -9,6 +9,7 @@
 import com.google.common.collect.Maps;
```
Comment (bsideup) [inline core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java]: Please also mention the change in CHANGELOG.md :)

**Comment by bsideup** [comment]
Great, thanks! :)

**Pull request merged**